src/content/docs/release-notes/docs-release-notes/docs-12-13-2024.mdx

```diff
@@ -0,0 +1,127 @@
+---
+subject: "Docs"
+releaseDate: '2024-12-13'
+version: 'version: November 29-December 13, 2024'
+---
+
+### New docs
+
+* [Logging with the iOS agent](https://docs.newrelic.com/docs/mobile-monitoring/new-relic-mobile-ios/configuration/ios-remote-logging/) introduces enhanced logging capabilities in the iOS agent, including log level configuration and automatic log collection features.
+* The [Kubernetes APM auto-attach](https://docs.newrelic.com/docs/kubernetes-pixie/kubernetes-integration/installation/k8s-agent-operator), formerly known as the Kubernetes agent operator, now streamlines full-stack observability for Kubernetes environments by automating APM instrumentation alongside Kubernetes agent deployment.
+* [Forward AWS services logs to New Relic](https://docs.newrelic.com/install/aws-logs/?service=aws_services) now include a new log integration method using a CloudFormation template to forward logs, customize triggers, and configure endpoints from New Relic One, in addition to existing methods.
```

Comment:
* [Forward AWS services logs to New Relic](https://docs.newrelic.com/install/aws-logs/?service=aws_services)  includes a new log integration method using a CloudFormation template to forward logs.

Reply:
Updated